docs/howto/writing-migrations.txt

```diff
@@ -86,11 +87,39 @@ the respective field according to your needs.
 * Add the field on your model with ``default=...`` and ``unique=True``
   arguments. In the example, we use ``uuid.uuid4`` for the default.
 
-* Run the :djadmin:`makemigrations` command.
+* Run the :djadmin:`makemigrations` command. This should generate a migration
+  with an ``AddField`` operation.
 
-* Edit the created migration file.
+* Generate two empty migration files for the app by running
+  ``makemigrations myapp --empty`` twice.
 
-  The generated migration class should look similar to this::
+* Copy the ``AddField`` operation from the auto-generated migration (the first
+  of the three new files) to the last migration and change ``AddField`` to
+  ``AlterField``. For example::
+
+        # -*- coding: utf-8 -*-
```

Comment:
I think we should use the [`:snippet:`](https://github.com/django/django/blob/master/docs/intro/tutorial01.txt#L350-L351) syntax to show explicit file names for the migrations